examples/sample-primitives-project/settings.gradle.kts

```diff
@@ -0,0 +1,12 @@
+pluginManagement {
+    repositories {
+        mavenCentral()
+        gradlePluginPortal()
+    }
+}
+
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.5.0"
```

Comment:
Never heard of this.  Is it supposed to be here?

Reply:
I think IntelliJ is creating projects with this as default? When I'm back home and at my computer I'll remove it to see if it has any effect. I did research it but don't remember what it is all about.

Reply:
@markATavail It is used to [resolve JVMs](https://github.com/gradle/foojay-toolchains). It is [mentioned](https://docs.gradle.org/current/userguide/toolchains.html#sub:download_repositories) by gradle.